### PR TITLE
added People's Antenna - PPL

### PR DESCRIPTION
**Pull request by peoplesantenna**
* added manufacturer antenna People's Antenna - PPL

### CODE REVIEW

File: src/makers/peoples/antennas.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const peoples = {
+  name:'"People\'s Antenna" - PPL',
```
Comment (tyler-whitman) [inline src/makers/peoples/antennas.ts:4]: There is a lint error here. It should be `  name: '"People\'s Antenna" - PPL',`

File: src/makers/index.ts [hunk 2]
```diff
@@ -68,6 +69,7 @@ export const AntennaModels = {
   ...syncrobit.antennas,
   ...kerlink.antennas,
   ...heltec.antennas,
+  ...peoples.antennas,
```
Comment (tyler-whitman) [inline src/makers/index.ts:72]: This should just be `...peoples`.

File: src/makers/peoples/antennas.ts (hunk 1 above)
Comment (tyler-whitman) [reply src/makers/peoples/antennas.ts:4]: You may also want to remove the `"` so that this will be in the proper alphabetical order.

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/makers/peoples/antennas.ts (hunk 1 above)
Comment (peoplesantenna) [reply src/makers/peoples/antennas.ts]: Fixed, thanks!

File: src/makers/index.ts (hunk 2 above)
Comment (peoplesantenna) [reply src/makers/index.ts]: Fixed

**Comment by peoplesantenna** [comment]
@tyler-whitman any idea when this will merge in? All the changes requested have been applied. Thank you!

**Comment by cokes518** [comment]
Thanks for this PR but we will not be merging this. It was a mistake to add the initial third-party antenna (HNTenna) to the app in the first place. The only antennas that should be in the dropdown should be from HIP-19 approved manufacturers. As such, any other custom/third-party antenna will be removed from the app and future third-parties should use the custom antenna dropdown field. Users should be informed enough to input the dbi themselves.

https://github.com/helium/hotspot-app/pull/1234

**Pull request closed without merging**